src/ert/gui/simulation/queue_emitter.py

```diff
@@ -50,10 +50,13 @@ def consume_and_emit(self) -> None:
                 continue
 
             # pre-rendering in this thread to avoid work in main rendering thread
-            if isinstance(event, FullSnapshotEvent) and event.snapshot:
+            if (
+                isinstance(event, FullSnapshotEvent)
```

Comment:
`if isinstance(event, (FullSnapshotEvent, SnapshotUpdateEvent)) and event.snapshot`